### ISSUE TITLE
Gravatar for Commenter

### ISSUE THREAD

**Issue by lucasisaza** [VenusInterns/BlogTemplate#103]
For people who leave a comment, use their emails to provide a Gravatar image.

**Comment by emanb29** [comment]
`Comment` authors are currently identified only by a provided `AuthorName`, and no email. Is part of the intention with this to add an email to `Comment`s?

**Comment by lucasisaza** [comment]
Yes, we originally had the email, but realized it would be a while until we would actually use it so it was removed. Part of the intention is to add the email.
